changelog.md

```diff
@@ -2,7 +2,13 @@
 
 ## UNRELEASED
 
-_Nothing yet..._
+**Breaking:**
+- bpk-component-spinner:
+  - Default spinner color is now blue-500
```

Comment:
Sorry - I know this is picky, but off the back of our talks about consistent changelog formatting, should `blue-500` be quoted as code?

Reply:
I opted to make this non-breaking so this has been removed.